src/app/person/person.cpp

```diff
@@ -32,11 +35,74 @@ std::string Person::getSurname() const
 
 void Person::setPesel(const std::string& pesel)
 {
-    // TODO Validation
     pesel_ = pesel;
 }
 
 std::string Person::getPesel() const
 {
     return pesel_;
 }
+
+bool Person::validatePesel(const std::string& pesel) const
+{
+    if (/*not validatePeselSize(pesel) or not validatePeselDate(pesel) or*/ not validateControlNumber(pesel))
```

Comment:
Why it is commented out? If methods are not necessary please then remove them

Reply:
Done